spec/models/family_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe Family do
+  describe "#create" do
+    it "is not valid without a last_name" do
+      new_family = Family.new(last_name: nil)
```

Comment:
I like that you are explicit here, but just calling `.new` would have suffice.

Also, look at using shoulda matchers for all the validations. Make life easier.